CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/plugins/blaze/BlazeActions.java

```diff
@@ -189,19 +345,76 @@ private void setBlazeProperties(final float size, final float opacity) {
         }
     }
 
+    private void updateCustomBlazes() {
+        for (int idx = 0; idx < MAXIMUM_CUSTOM_BLAZE_COLORS; idx++) {
+            this.recentCustomBlazeItems[BlazeActions.getRecentCustomColors().size()].setVisible(false);
+            this.presetCustomBlazeItems[BlazeActions.getPresetCustomColors().size()].setVisible(false);
+        }
+
+        customColorMenu.setEnabled(BlazeActions.getRecentCustomColors().size() > 0 || BlazeActions.getPresetCustomColors().size() > 0);
+        recentColorMenu.setEnabled(BlazeActions.getRecentCustomColors().size() > 0);
+        presetColorMenu.setEnabled(BlazeActions.getPresetCustomColors().size() > 0);
```

Comment:
should be able to make use of `.isEmpty()` rather than testing `size > 0` 
e.g. `!BlazeActions.getRecentCustomColors().isEmpty()`

Could also make the 2 primary boolean functions being used here as variables so that you only have to calculate the values once (and may make it more readable)